downloads/zh_CN/github-git-cheat-sheet.md

```diff
@@ -16,39 +16,39 @@ leadingpath: ../../../
 [git-scm.com](https://git-scm.com)
 
 ## 配置工具
-对所有本地仓库的用户信息进行配置
+##### 对所有本地仓库的用户信息进行配置。
```

Comment:
@lynkas I don't think this is appropriate. The translation should respect the original format. Besides you shouldn't use h4 directly under h2. If you want an emphasis, `**` is to be used.

Reply:
It appears https://github.com/github/training-kit/pull/752 is open to address this 👍 Thank you @imba-tjd!